www/js/control/SettingRow.tsx

```diff
@@ -33,9 +39,12 @@ const SettingRow = ({ textKey, iconName, action, desc, switchValue, descStyle }:
       style={styles.item(colors.surface)}
       title={t(textKey as any)}
       titleStyle={styles.title}
-      description={desc}
+      description={descriptionText}
```

Comment:
and this. The related commit is: https://github.com/sebastianbarry/e-mission-phone/commit/69e3bee47e0aa4f4623f551ab7e14721182be8dd which does not have any explanation.

Reply:
Resolved in https://github.com/e-mission/e-mission-phone/pull/1118/commits/3090f33606c18a92af570483e0e7a837ba2086f2